### ISSUE TITLE
How to set upload automatically when taking image capture, but not when doing video?

### ISSUE THREAD

**Issue by MostHated** [ShareX/ShareX#4535]
Hey there,
    I have ShareX set to automatically upload to Imgur on image capture, but I don't want it to auto-upload anywhere when it has completed doing video capture so I can edit it first, then upload that to Imgur as well (Imgur doesn't seem to be on the list for video upload either? I always upload my mp4's to be converted to gifv). 

  Currently, it keeps trying to upload to dropbox, which I don't have or want to use, but I can't seem to select nothing/none or disable video auto-upload. Is there a setting for that in which I am just not seeing?

Thanks,
-MH

**Comment by MostHated** [comment]
Alternatively, it would be great to be able to upload recorded video clips directly to imgur, as that is where I always upload them already as they automatically convert mp4 to gifv.

**Comment by Fuzzqt** [comment]
Click Hotkey Settings within the Sharex GUI > Click the Cogwheel for your specific hotkey that has to do with uploading to Dropbox, or make a separate hotkey (one for online and one for offline) 

After Capture: Save image to File

After Upload: (leave this blank)

![ShareX_OnQHKhAHrA](https://user-images.githubusercontent.com/47482485/73828672-afb68c00-47cf-11ea-915c-bdb17504b399.png)

**Comment by MostHated** [comment]
Hey there, I appreciate your reply. I don't recall what I did offhand, but I ended up figuring it out 2-3 weeks ago, so now it just opens a small window afterward that lets me open the file or folder from there.Not ideal, but better than it was. I don't think my method of capture is available in the hotkey list, I don't believe it's actually a hotkey. It is triggered via an interface event (simply clicking on the little taskbar icon once enables the region capture, so I just drag a box and it starts recording. I then click the little red recording icon to stop it.)
